src/gtest/test_transaction_builder.cpp

```diff
@@ -94,7 +94,7 @@ TEST(TransactionBuilder, TransparentToSapling)
     // Create a shielding transaction from transparent to Sapling
     // 0.0005 t-ZEC in, 0.0004 z-ZEC out, 0.0001 t-ZEC fee
     auto builder = TransactionBuilder(consensusParams, 1, &keystore);
-    builder.AddTransparentInput(COutPoint(), scriptPubKey, 50000);
+    builder.AddTransparentInput(COutPoint(uint256S("1234"), 0), scriptPubKey, 50000);
```

Comment:
This was necessary because a transaction with a single blank `COutPoint()` in `tx.vin` creates what the consensus rules treat as a coinbase transaction, which was unintentional in this test and IIRC causing this test to be broken by the new consensus rules.